devenv/direnvrc

```diff
@@ -102,7 +102,17 @@ _devenv_watches() {
 use_devenv() {
   _nix_direnv_preflight
 
+  # First parameter is the flake expression, defaults to .
   flake_expr="${1:-.}"
```

Comment:
We can probably remove this and `flake_dir`. This got inherited from `nix_direnv` at some point.